### PR TITLE
README for SWIFT/T (#37)

### PR DESCRIPTION
**Pull request by j-woz**

### CODE REVIEW

File: docker/swift-t/README.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+== Method
+
+=== Usage of Base
+
+This image relies on several packages from the ExaWorks SDK Base (Centos 7), particularly the C tools and OpenMPI.
+
+=== Dependencies
+
+Dependencies are considered in the shell script install-deps.sh .
```
Comment (dongahn) [inline docker/swift-t/README.adoc:22]:
```suggestion
Dependencies are considered in the shell script `install-deps.sh`.
```

File: docker/swift-t/README.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+ZSH and SWIG are installed from Yum.
+
+Swift/T relies on Tcl 8.6, which is not supported in Centos 7, so we build that manually.
+
+=== Build
+
+Swift/T is built from source in install-swift-t.sh .
```
Comment (dongahn) [inline docker/swift-t/README.adoc:30]:
```suggestion
Swift/T is built from source in `install-swift-t.sh`.
```

File: docker/swift-t/README.adoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+This is a standard build from GitHub.  sed is used to populate the Swift/T settings file before the build.
```
Comment (dongahn) [inline docker/swift-t/README.adoc:32]:
```suggestion
This is a standard build from the Swift/T GitHub repo. `sed` is used to populate the Swift/T settings file before the build.
```

File: docker/swift-t/README.adoc [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+See the following link for more information about compiling Swift/T: http://swift-lang.github.io/swift-t/guide.html#install_source
```
Comment (dongahn) [inline docker/swift-t/README.adoc:34]:
```suggestion
You can find more information about compiling Swift/T [here](http://swift-lang.github.io/swift-t/guide.html#install_source)
```

File: docker/swift-t/README.adoc [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+== Tests
+
+Currently, a small test suite is run from shell script test-sanity.sh .
```
Comment (dongahn) [inline docker/swift-t/README.adoc:38]:
```suggestion
Currently, a small test suite is run from shell script `test-sanity.sh`.
```

**Review by dongahn — CHANGES_REQUESTED** [review]
Superb. Super nit suggestions are in-lined below.

**Review by mturilli — COMMENTED** [review]
Maybe add a note+link to Swift/T examples and overall documentation? Otherwise, LGTM.

**Comment by j-woz** [comment]
Good idea, I will add that.

**Comment by j-woz** [comment]
Ok those items are in as "Further information"

**Review by mturilli — APPROVED** [review]
LGTM!

**Comment by j-woz** [comment]
Sure.

**Comment by j-woz** [comment]
Ok- I think 31a6bf78 is the single commit for this README

**Comment by dongahn** [comment]
Thanks @j-woz. It seems this PR has all other changes you are currently making? Did you mean to create a PR with a single commit instead?

Probably the easiest way to do this is to create a branch in your fork out of the upstream master and add that single commit and do force a push, I think.

**Pull request closed without merging**